### DISCUSSION TITLE
Private hosting: VPS Spec

### DISCUSSION THREAD

**Discussion by jnnngs** [searx/searx#862] [General]
Hi,

What VPS spec is recommended is your hosting searx yourself?

Thanks in advance!

**Comment by virtadpt** [comment]
I'm running an instance on a Digital Ocean VPS (4GB RAM/60GB disk space/4 CPUs) (I think) behind Nginx from a GNU Screen session.  I've no doubt that I could run it on the 512 MB RAM version without any trouble.

**Comment by kvch** [comment]
I run searx on a Digital Ocean instance with 512 MB RAM, 1 core, 20 GB SSD disk using nginx and tmux. So far I had no trouble, searx runs flawlessly.

**Comment by jnnngs** [comment]
Thanks dudes. I'll take it for a spin :)

**Comment by mangeurdenuage** [comment]
There's a new vps provider with only libre hardware (deblobed AMD/PSP bios) if anyone interested.
https://store.vikings.net/
